index.html

```diff
@@ -5663,7 +5663,21 @@ <h3>Language and Base Direction</h3>
   "@id": "https://www.w3.org/2018/credentials/examples#title"
 }
         </pre>
+      </section>
+
+      <section class="informative">
+        <h3>Providing Default Language and Direction</h3>
 
+        <p>
+A default language and direction MAY be provided for the entire credential.
+However, doing so introduces a higher demand on creating a compliant processor
+and fully compliant context files. Consequently, using default language and
+direction should be limited to scenarios where JSON-LD expansion-based
+transformation can be assured. See the
+<a href="https://www.w3.org/TR/json-ld11/#string-internationalization">String
```

Comment:
I think we can do something like this instead (please check the syntax, don't know if it's accurate):

```suggestion
<a date-cite="JSON-LD#string-internationalization">String
```